### PR TITLE
Add a new method searchBucketVersions

### PR DESCRIPTION
**Pull request by ChengYanJin**

### CODE REVIEW

**Review by XinLi0207 — APPROVED** [review]

**Comment by JBWatenbergScality** [comment]
Shall we add a test for searchBucketVersions api ?

File: zenko-2018-07-11-xml.api.json [hunk 1]
```diff
@@ -107,6 +107,92 @@
...
+                    },
+                    "ExpectedBucketOwner": {
+                        "location": "header",
+                        "locationName": "x-amz-expected-bucket-owner"
+                    },
+                    "Versions": {
+                        "location": "querystring",
+                        "locationName": "versions",
+                        "type": "boolean"
+                    },
```
Comment (JBWatenbergScality) [inline zenko-2018-07-11-xml.api.json]: I find it a bit strange to have both an Api name and a flag versions. What happens if we call `searchBucketVersions` with version being `false` ?

File: test/ZenkoClient.test.js [hunk 2]
```diff
@@ -222,12 +257,48 @@ const expectedSearchResult = {
...
+    DeleteMarker: [{
+        Key: 'meeting-83545816150.ics',
+        IsLatest: true,
+        LastModified: new Date('2022-04-14T10:55:14.196Z'),
+        Owner: {
+            DisplayName: 'orbit',
+            ID: ownerId,
+        },
+        VersionId: '39383335303036363238353739383939393939395247303030303132342e36',
+    }],
```
Comment (JBWatenbergScality) [inline test/ZenkoClient.test.js]: nit: I would mutualize those expected values with the one mocked

**Review by alexanderchan-scality — APPROVED** [review]

**Pull request merged**